README.md

```diff
@@ -70,6 +70,14 @@ bin/fluent-bit -i cpu -o stdout -f 1
 
 If you are interested into more details, please refer to the [Build & Install](https://docs.fluentbit.io/manual/installation/sources/build-and-install) section.
 
+#### Requirements
+
+- CMake >= 3.0
+- Flex
+- Bison
+- YAML headers
+- OpenSSL headers
```

Comment:
We also need YAML/OpenSSL libraries for linking, no? Maybe change these two lines to

```
YAML library/headers
OpenSSL library/headers
```

Reply:
Thank you for pointing it.
I added them.